_unittest_ironpython/test_Simplorer.py

```diff
@@ -0,0 +1,23 @@
+
+from conf_unittest import test_generator, PytestMockup
+import os
+
+test_filter = "test_"
+
+test_name = os.path.basename(__file__).replace(".py", "")
+mymodule = __import__('_unittest.' + test_name, fromlist=['TestClass'])
+test_obj = mymodule.TestClass()
+
+class TestSequenceFunctionsGenerate(PytestMockup):
+    @classmethod
+    def setUpClass(cls):
+        test_obj.setup_class()
+
+    @classmethod
+    def tearDownClass(cls):
+        test_obj.teardown_class()
```

Comment:
This appears to be shared across several of these unit tests.  Can we consider a refactor to have it within a fixture/shared module?

Reply:
@vossleon next PR?